transport/http/constants.go

```diff
@@ -77,8 +81,38 @@ const (
 
 	// An error occurred. The response body contains an application header.
 	ApplicationErrorStatus = "error"
+
+	// AcceptTrue is the true value for accept headers.
+	AcceptTrue = "true"
+
+	// AcceptFalse is the false value for accept headers.
+	AcceptFalse = "false"
 )
 
 // ApplicationHeaderPrefix is the prefix added to application header keys to
 // send them in requests or responses.
 const ApplicationHeaderPrefix = "Rpc-Header-"
+
+func applicationStatusValue(isApplicationError bool) string {
+	if isApplicationError {
+		return ApplicationErrorStatus
+	}
+	return ApplicationSuccessStatus
+}
+
+func fromApplicationStatusValue(applicationStatusValue string) bool {
+	// TODO(pedge): should we ignore if not equal to success and error?
```

Comment:
Good question. I think we can interpret anything that’s not absent or "success" as "error".

Reply:
I like moving to the model I had in https://github.com/yarpc/yarpc-go/pull/1318, what do you think?

Reply:
Moved to only having a true value.